modules/multi-architecture-migrating-cluster.adoc

```diff
@@ -0,0 +1,31 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/cluster-tasks.adoc
+:_content-type: PROCEDURE
+[id="multi-architecture-migrating-cluster_{context}"]
+
+= Migrating to a cluster with multi-architecture compute machines 
+
+You can use the following procedure to upgrade existing single-architecture cluster to one that supports multi-architecture compute machines.
+
+[IMPORTANT]
+====
+Before you migrate from a from a single-architecture cluster, ensure that the Cluster Samples Operators is disabled. You can remove the Operator with the following command: 
+[source,terminal]
+----
+$ oc patch clusterversion/version --type merge -p '{"spec":{"capabilities":{"additionalEnabledCapabilities":["baremetal", "marketplace", "Console", ..... ]}}}'
```

Comment:
do the changes stick after the upgrade?

Reply:
mmm, we cannot remove an already enabled capability, can we?

Reply:
yeah i wouldn't mention it at all in this case. It doesn't hurt to have those samples on the cluster and this is just a tedious workaround to remove that which is not necessary

Reply:
Gotcha! Should I remove the warning entirely or just mention something about the Samples operator not being fully functional?

Reply:
there should be mention that samples is unsupported in the sense that the imagestreams generated are not entirely usable and that there is an option when installing a cluster - to exclude that capability.